### DISCUSSION TITLE
How should guilds function?

### DISCUSSION THREAD

**Discussion by Volte6** [GoMudEngine/GoMud#87] [Q&A]
I admit I'm a little bit on the fence about guilds, mostly because they are a feature that really exists for games with a large active player base, and I imagine a typical mud being a handful of people these days.

However, it's worth thinking about... especially if there's a way to make a guild interesting even with a low player count.

One idea I'm toying with is the idea of there being a central guild hall (found via a secret entrance or just a certain area) in each area... so Frostfang would have one, the dark forest would have one, etc... 

Any established guild who exerts the most influence over the area in a given time period gets exclusive access to the guild hall of the area, which would provide them with a base of operations, and some guild-hall specific benefits (exclusive item storage, map of the entire area showing where players or mobs are, etc).

They could exert influence by any number of things... killing the most mobs, tagging the most areas, or even just spending time in the areas the most cumulatively.

I'm open to ideas though.

**Comment by windu-ant** [comment]
I really like this idea about influence on areas. 

The favorite way I've seen guilds handled in MUDs is giving you some sort of perk for being in the guild, while also offering some unique features or mechanics.  For example, one guild might give you access to a special shop with the best healing foods, while another one might let you store extra MP into a charm. Each guild could have a way to go about earning currency. One might be donating items, while another might be returning orc and goblin ears. In your example, guilds could get increased benefits for controlling the most areas by tagging or killing. These guilds aren't really owned by players, but are static guilds you can join that are "owned" by NPC's. This would also allow the admin to control the amount of possible Guilds dependent on your player base.

This would be separate from something like NPC reputation or factions, in the fact that you could only be a part of one at a time, and it might also offer other community benefits like a channel only for players of that guild. 

Could also have player-ran vs guild-ran groups that provide different mechanics. Clans for one and Guilds or whatever naming convention for the other. Perhaps having Clans be player ran and controlling the influence of things like mentioned above, and Guilds could be more PvE related - collecting pelts, orc ears, <insert item here> for currency. 

While this still would be best with a large amount of players, you could still have a few players work together to reach certain goals. If you had resets weekly, or some sort of decay, you could have a cooperative experience to try to unlock the best things from a Guild before the reset happened. One MUD I played would give the guild with the most claimed rooms access to the best armor in the game, and then on reset you would have to claim all the rooms again if you wanted it.

**Reply by Volte6** [reply]
I was definitely thinking of "player clans" and "guilds" as interchangeable terms - but I do like the idea of special NPC run guilds that operate similarly.

I also like the idea of stretch goals for guilds to reach, maybe a way to "level up" their guild progressively to increase its capability.
